src/Microsoft.Performance.Toolkit.Engine/Engine.cs

```diff
@@ -805,6 +808,22 @@ protected void ThrowIfProcessed()
             }
         }
 
+        /// <summary>
+        ///     Creates a logger for the given type.
+        /// </summary>
+        /// <param name="type">
+        ///     The <c>Type</c> for which the logger is created.
+        /// </param>
+        /// <returns>
+        ///     An instance of <see cref="ILogger"/>.
+        /// </returns>
+        protected ILogger CreateLogger(Type type)
+        {
+            Debug.Assert(!this.isDisposed);
```

Comment:
is this line redundant with the line below?

Reply:
No, this can make it easier to find while debugging.